src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -64,6 +64,9 @@ public static function load()
 
             $info = curl_getinfo($ch);
             $sanitizedUrl = Urls::sanitize($info['url']);
+            if ($globalConfig->isHttpClientSplitByDomain()) {
```

Comment:
What's the motivation for keeping `$span->setTag(Tag::SERVICE_NAME, 'curl');`? In case something in between fails we want to at least set `curl` as the service name?

Reply:
fixed, thanks for spotting it.